src/readers/morphologyHDF5.cpp

```diff
@@ -430,8 +411,9 @@ void MorphologyHDF5::_readPerimeters(int firstSectionOffset) {
                                                       perimeters.end());
     } catch (...) {
         if (_properties._cellLevel._cellFamily == FAMILY_GLIA)
-            throw MorphioError("No empty perimeters allowed for glia "
-                             "morphology");
+            throw MorphioError(
+                "No empty perimeters allowed for glia "
+                "morphology");
```

Comment:
```suggestion
            throw MorphioError("No empty perimeters allowed for glia morphology");
```
Seems like some weird formatting stuff here…